src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    Boolean update(MovieSession movieSession);
```

Comment:
Consider changing the return type of the `update` method to `void` or returning the updated `MovieSession` entity if applicable. Returning a `Boolean` might not be necessary if exceptions are used to indicate failure.